sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java

```diff
@@ -70,41 +70,12 @@ public CosmosBulkExecutionOptions() {
         this(null, null, null);
     }
 
-    /**
-     * The maximum batching size for bulk operations. This value determines number of operations executed in one
-     * request. There is an upper limit on both number of operations and sum of size of operations. Any overflow is
-     * internally retried.
-     *
-     * Another instance is: Currently we support a max limit of 200KB, and user select batch size to be 100 and individual
-     * documents are of size 20KB, approximately 90 operations will always be retried. So it's better to choose a batch
-     * size of 10 here if user is aware of there workload. If sizes are totally unknown and user cannot put a number on it
-     * then retries are handled, so no issues as such.
-     *
-     * If the retry rate exceeds `getMaxMicroBatchInterval` the micro batch size gets dynamically reduced at runtime
-     * @return micro batch size
-     */
-    int getMaxMicroBatchSize() {
-        return maxMicroBatchSize;
+    int getInitialMicroBatchSize() {
```

Comment:
maybe add some java doc for this property?

Reply:
It is intenral - intentionally not added - what else than the name already tells?